shaper/libs/parser.py

```diff
@@ -176,6 +176,13 @@ def write(self, data, path):
                 represent_unicode,
             )
 
+        # string representer for multi line issue
+        def str_presenter(dumper, data):
```

Comment:
I would suggest to move this function to `shaper.libs.loader.py` module

Reply:
done